api/queries/channel/metaData.js

```diff
@@ -3,17 +3,14 @@ import type { GraphQLContext } from '../../';
 import type { DBChannel } from 'shared/types';
 
 export default async (
-  { id }: DBChannel,
+  { id, memberCount }: DBChannel,
   _: any,
   { loaders }: GraphQLContext
 ) => {
-  const [threads, members] = await Promise.all([
-    loaders.channelThreadCount.load(id),
-    loaders.channelMemberCount.load(id),
-  ]);
+  const [threads] = await Promise.all([loaders.channelThreadCount.load(id)]);
 
   return {
     threads: threads ? threads.reduction : 0,
-    members: members ? members.reduction : 0,
+    members: memberCount || 1,
```

Comment:
Should probably fallback to `loaders.channelMemberCount.load` just to make sure?